toar/toar_planetscope.ipynb

```diff
@@ -0,0 +1,474 @@
+      "pending: 2                                                                      \u001b[39;49m\n",
+      "\n",
+      "\u001b[2J\u001b[1;1H\u001b[30;47mactivating: 0            complete: 0              downloaded: 0.00MB            \n",
+      "downloading: 2           elapsed: 3               paging: False                 \n",
+      "pending: 0                                                                      \u001b[39;49m\n",
+      "\n",
+      "\u001b[2J\u001b[1;1H\u001b[30;47mactivating: 0            complete: 0              downloaded: 11.80MB           \n",
+      "downloading: 2           elapsed: 4               paging: False                 \n",
+      "pending: 0                                                                      \u001b[39;49m\n",
+      "\n",
+      "\u001b[2J\u001b[1;1H\u001b[30;47mactivating: 0            complete: 0              downloaded: 50.37MB           \n",
+      "downloading: 2           elapsed: 5               paging: False                 \n",
+      "pending: 0                                                                      \u001b[39;49m\n",
+      "\n",
+      "\u001b[2J\u001b[1;1H\u001b[30;47mactivating: 0            complete: 0              downloaded: 126.99MB          \n",
+      "downloading: 2           elapsed: 6               paging: False                 \n",
+      "pending: 0                                                                      \u001b[39;49m\n",
+      "\n",
+      "\u001b[2J\u001b[1;1H\u001b[30;47mactivating: 0            complete: 0              downloaded: 126.99MB          \n",
+      "downloading: 2           elapsed: 7               paging: False                 \n",
+      "pending: 0                                                                      \u001b[39;49m\n",
+      "\n",
+      "\u001b[2J\u001b[1;1H\u001b[30;47mactivating: 0            complete: 0              downloaded: 126.99MB          \n",
+      "downloading: 2           elapsed: 7               paging: False                 \n",
+      "pending: 0                                                                      \u001b[39;49m\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!planet data download --item-type PSScene4Band --asset-type analytic,analytic_xml --string-in id 20170623_180038_0f34"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Congratulations!** You now have two files in your download directory: `20170623_180038_0f34_3B_AnalyticMS.tif` and `20170623_180038_0f34_3B_AnalyticMS_metadata.xml`. The first file is a GeoTIFF, the image you requested with spatial reference data embedded. The second file is a metadata file for that image that includes the data you'll need to convert from radiance to reflectance. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2. Extract the Data from Each Color Band "
```

Comment:
Again, you could skip the download step and refer users to the NDVI guide. Your guide could start with "extract data from the spectral bands" or it could start with Extract the coefficients from the metadata.